### PR TITLE
changed the readme file

### PR DESCRIPTION
**Pull request by macbarnes04**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,91 @@
...
+```
+
+| Parameter | Type     | Description                       |
+| :-------- | :------- | :-------------------------------- |
+| `id`      | `string` | **Required**. Id of item to fetch |
+
+#### add(num1, num2)
+
+Takes two numbers and returns the sum.
+
```
Comment (ganning127) [inline README.md:67]: Delete this section; we don't have an API

File: README.md [hunk 2]
```diff
@@ -1 +1,91 @@
...
+## Media Toolkit - Color Codes
+
+| Color             | Hex                                                                |
+| ----------------- | ------------------------------------------------------------------ |
+| Example Color | ![#0a192f](https://via.placeholder.com/10/0a192f?text=+) #0a192f |
+| Example Color | ![#f8f8f8](https://via.placeholder.com/10/f8f8f8?text=+) #f8f8f8 |
+| Example Color | ![#00b48a](https://via.placeholder.com/10/00b48a?text=+) #00b48a |
+| Example Color | ![#00d1a0](https://via.placeholder.com/10/00b48a?text=+) #00d1a0 |
+
```
Comment (ganning127) [inline README.md:76]: Delete

File: README.md [hunk 3]
```diff
@@ -1 +1,91 @@
...
+Answer 1
+
+#### Question 2
+
+Answer 2
+
+
+## Feedback
+
+If you have any feedback, please reach out to us at fake@fake.com
```
Comment (ganning127) [inline README.md:91]: Replace email with: info@bitproject.org

File: README.md [hunk 4]
```diff
@@ -1 +1,91 @@
...
+#### Question 1
+
+Answer 1
+
+#### Question 2
+
+Answer 2
+
+
+## Feedback
```
Comment (ganning127) [inline README.md:89]: FAQ: 

**Are your programs free?**
Yes! Both our mentorship program and global student community are free for all members to join. However, the mentorship program is highly selective and requires an application.

**How can I contribute?**
Email us at info@bitproject.org and we'll be happy to work with you!

File: README.md [hunk 5]
```diff
@@ -1 +1,91 @@
...
+- Fifi [saved money shopping on Amazon](https://www.bitproject.org/projects/saving-money-with-serverless) with Serverless 🛒
+
+
+## Meet the Team!
+
+- 💙 Shreya Gupta, Founder (Operations) [Github Link](https://www.github.com/octokatherine)
+
+- 💙 Daniel Kim, Founder (Technology) [Github Link](https://www.github.com/octokatherine)
+- 💙 Ganning Xu, Director of Engineering [Github Link](https://www.github.com/octokatherine)
+- 💙 Emily Chen, Director of Engineering [Github Link](https://www.github.com/octokatherine)
```
Comment (ganning127) [inline README.md:36]: https://github.com/emsesc

File: README.md [hunk 6]
```diff
@@ -1 +1,91 @@
...
+- Jakub [analyzed vehicle telemetry data](https://www.bitproject.org/projects/vehicle-telemetry-sigma-link) in real-time 🚗
+- Kaley developed [TimeBlocker](https://www.bitproject.org/projects/saving-time-with-timeblocker) and Boosted Productivity 📚
+- Fifi [saved money shopping on Amazon](https://www.bitproject.org/projects/saving-money-with-serverless) with Serverless 🛒
+
+
+## Meet the Team!
+
+- 💙 Shreya Gupta, Founder (Operations) [Github Link](https://www.github.com/octokatherine)
+
+- 💙 Daniel Kim, Founder (Technology) [Github Link](https://www.github.com/octokatherine)
```
Comment (ganning127) [inline README.md:34]: https://github.com/lazyplatypus

File: README.md [hunk 7]
```diff
@@ -1 +1,91 @@
...
+- 💙 Ganning Xu, Director of Engineering [Github Link](https://www.github.com/octokatherine)
```
Comment (ganning127) [inline README.md:35]: https://github.com/ganning127

File: README.md [hunk 8]
```diff
@@ -1 +1,91 @@
...
+
+# Bit Project Website
+
+This GitHub repo is the home 🏠 of the Bit Project website. 
+Here at the Bit Project We equip students with technical 
+superpowers by providing cutting-edge, open source learning 
+experiences to prepare students for careers in tech. 
+
+
+## Bit Project Sponsors
```
Comment (ganning127) [inline README.md:12]: Change to: "Serverless Camp Sponsors"

File: README.md [hunk 9]
```diff
@@ -1 +1,91 @@
...
+
+## Meet the Team!
+
+- 💙 Shreya Gupta, Founder (Operations) [Github Link](https://www.github.com/octokatherine)
+
+- 💙 Daniel Kim, Founder (Technology) [Github Link](https://www.github.com/octokatherine)
+- 💙 Ganning Xu, Director of Engineering [Github Link](https://www.github.com/octokatherine)
+- 💙 Emily Chen, Director of Engineering [Github Link](https://www.github.com/octokatherine)
+- 💙 Misha Patel, Director of Partnerships [Github Link](https://www.github.com/octokatherine)
+- 💜 Beatrix Cendana, Student Community Manager [Github Link](https://www.github.com/octokatherine)
```
Comment (ganning127) [inline README.md:38]: https://github.com/beatrixcendana

**Review by ganning127 — CHANGES_REQUESTED** [review]
Requested changes are in comments!

File: README.md [hunk 10]
```diff
@@ -1 +1,91 @@
...
+
+- Grace built [EcoMaps](https://www.bitproject.org/projects/reducing-carbon-emissions) and reduced carbon emissions 🌱
+- Jakub [analyzed vehicle telemetry data](https://www.bitproject.org/projects/vehicle-telemetry-sigma-link) in real-time 🚗
+- Kaley developed [TimeBlocker](https://www.bitproject.org/projects/saving-time-with-timeblocker) and Boosted Productivity 📚
+- Fifi [saved money shopping on Amazon](https://www.bitproject.org/projects/saving-money-with-serverless) with Serverless 🛒
+
+
+## Meet the Team!
+
+- 💙 Shreya Gupta, Founder (Operations) [Github Link](https://www.github.com/octokatherine)
```
Comment (ganning127) [inline README.md:32]: https://github.com/shreythecray

File: README.md [hunk 11]
```diff
@@ -1 +1,54 @@
...
+- 💙 Daniel Kim, Founder (Technology) [Github Link](https://github.com/lazyplatypus)
+- 💙 Ganning Xu, Director of Engineering [Github Link](https://github.com/ganning127)
+- 💙 Emily Chen, Director of Engineering [Github Link](https://github.com/emsesc)
+- 💙 Misha Patel, Director of Partnerships [Github Link](https://www.github.com/octokatherine)
+- 💜 Beatrix Cendana, Student Community Manager [Github Link](https://github.com/beatrixcendana)
+
+
+## FAQ
+
+#### Are your programs free?
```
Comment (ganning127) [inline README.md:43]: Can you make these a dropdown? Here's the format for it:
```
<details>
  <summary>Want to ruin the surprise?</summary>
  <br>
  Well, you asked for it!
</details>
```

**Example:**
<details open>
<summary>Want to ruin the surprise?</summary>
<br>
Well, you asked for it!
</details>

Comment (ganning127) [reply README.md:43]: Here's the source! https://gist.github.com/citrusui/07978f14b11adada364ff901e27c7f61

File: pages/index.js [hunk 12]
```diff
@@ -36,7 +36,7 @@ export default function HomePage() {
       </Head>
       <Navbar />
       <Landing
-        heading="We equip students with technical superpowers"
+        heading="test"
```
Comment (ganning127) [inline pages/index.js:39]: Can you change this back to: "We equip students with technical superpowers" from "test" LOL

File: README.md [hunk 13]
```diff
@@ -1 +1,54 @@
...
+
+
+## Meet the Team!
+
+- 💙 Shreya Gupta, Founder (Operations) [Github Link](https://github.com/shreythecray)
+
+- 💙 Daniel Kim, Founder (Technology) [Github Link](https://github.com/lazyplatypus)
+- 💙 Ganning Xu, Director of Engineering [Github Link](https://github.com/ganning127)
+- 💙 Emily Chen, Director of Engineering [Github Link](https://github.com/emsesc)
+- 💙 Misha Patel, Director of Partnerships [Github Link](https://www.github.com/octokatherine)
```
Comment (ganning127) [inline README.md:37]: Remove Misha's GitHub, she hasn't sent me her's yet.

File: README.md [hunk 14]
```diff
@@ -1 +1,54 @@
...
+
+- Grace built [EcoMaps](https://www.bitproject.org/projects/reducing-carbon-emissions) and reduced carbon emissions 🌱
+- Jakub [analyzed vehicle telemetry data](https://www.bitproject.org/projects/vehicle-telemetry-sigma-link) in real-time 🚗
+- Kaley developed [TimeBlocker](https://www.bitproject.org/projects/saving-time-with-timeblocker) and Boosted Productivity 📚
+- Fifi [saved money shopping on Amazon](https://www.bitproject.org/projects/saving-money-with-serverless) with Serverless 🛒
+
+
+## Meet the Team!
+
+- 💙 Shreya Gupta, Founder (Operations) [Github Link](https://github.com/shreythecray)
```
Comment (ganning127) [inline README.md:32]: Can you make all of these formatted as:
💙 Shreya Gupta ([@shreythecray](https://github.com/shreythecray)), Founder (Operations) 

Make sure to replace what's in the parenthesis with their github username of course

**Review by ganning127 — CHANGES_REQUESTED** [review]
Looks great! I just requested a few more small changes 😄

File: pages/index.js (hunk 12 above)
Comment (macbarnes04) [reply pages/index.js:39]: I changed it back in our zoom, its not showing up as test in any of the branches I have

**Review by ganning127 — APPROVED** [review]

**Pull request merged**